src/molecules/Teaser/Teaser.js

```diff
@@ -49,11 +48,7 @@ const Teaser = ({
               <Heading level={headingLevel}>{title}</Heading>
             </div>
             <div className="teaser_meta">
-              {publishDate && (
-                <time className="teaser_time" dateTime={publishDate}>
-                  {format(publishDate, 'MMMM D, YYYY')}
-                </time>
-              )}
+              {publishDate && publishDate}
```

Comment:
I can't remember seeing or needing a "check if I exist and, if so, show me" command in other React projects. Is this necessary?

Reply:
You're right, I don't need the check for this; react just won't show it if it doesn't exist. Since there's no html inside this anymore we don't need the check. I'll change that and merge.